api/spec/requests/spree/api/v2/storefront/cart_spec.rb

```diff
@@ -100,4 +100,44 @@
       end
     end
   end
+
+  describe 'cart#empty' do
+    context 'without existing order' do
+      it 'returns status code 404' do
+        headers = { 'Authorization' => "Bearer #{token.token}" }
+        post '/api/v2/storefront/cart/empty', headers: headers
+
+        expect(response.status).to eq(404)
+      end
+    end
+
+    context 'with existing order and line item' do
+      let!(:order) { create(:order, user: user) }
+      let!(:line_item) { create(:line_item, order: order) }
+
+      it 'empties the order' do
+        headers = { 'Authorization' => "Bearer #{token.token}" }
+        post '/api/v2/storefront/cart/empty', headers: headers
+
+        expect(response.status).to eq(200)
+
+        expect(order.line_items.count).to eq(0)
```

Comment:
@WitekL please use shared examples here - those tests for signed in user and guest user are basically the same